src/main/java/io/github/jhipster/config/JHipsterDefaults.java

```diff
@@ -36,6 +36,8 @@
 
         Version version = Version.V_1_1;
 
+        boolean useUndertowUserCipherSuitesOrder = false;
```

Comment:
why not enable it by default? I would put this to true

Reply:
Ok, you're right like this it will be automated